data/jibauni/jibauni 0 kanlanmo transport.txt

```diff
@@ -0,0 +1,628 @@
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+mission "Jibauni: Kanlanmo 0"
+	name "Take Kanlanmo to <planet>"
+	description "Kanlanmo, who is a Jibauni in the shape of a Tyrannosaurus Rex, has asked to go to <planet> on your ship. He will explain more once you get there."
+	landing
+	source "Baitoon"
+	destination "Taltahard"
+	on offer
+		conversation
+			`Your landing location is a large set of old landing pads in the middle of a city, as there doesn't seem to be anywhere else available for spaceships to arrive. The city is filled with dinosaurs in the shape of the mythical Tyrannosaurus Rex, although here smaller then in old Earth fables. Soon after you land, one of these dinosaurs, wearing a dark green cloak, approaches your ship.`
+			choice
+				`	(Go out and meet it.)`
+					goto meeting
+				`	(Leave now, before something bad happens.)`
+					depart
+				`	(Stay in <ship>.)`
+			`	The dinosaur opens its mouth, and a moment later a growling sound rumbles through the air. Then you hear the words, "Could you exit your ship to consult with me?"`
+				to display
+					not "language: Coalition"
+				goto initialchoice1
+			action
+				set "jibauni knowledge: the Jibauni have Heliarch translators"
+			`	The dinosaur opens its mouth to say something, and you hear a growling sound. Then you notice that it has a Coalition translator box around its neck. Before you can do anything the box says, "Could you exit your ship to consult with me?"`
+			label initialchoice1
+			choice
+				`	(Go outside of the <ship> and meet the strange beast.)`
+					goto meeting
+				`	(Leave the planet.)`
+					defer
+			label meeting
+			`	"Hello, fair stranger," it says. "Is there a way I may serve you on this delightful day?"`
+			label meetingquestions
+			choice
+				`	"My name is <first> <last>, and I come in peace."`
+					goto lastisknown
+				`	"How do you have a Coalition translator?"`
+					to display
+						has "jibauni knowledge: the Jibauni have Heliarch translators"
+						not "asked about the Jibauni translators"
+					goto cotrans
+				`	"Hi, weirdo. What's up?"`
+					goto rude
+				`	(Shoot the alien with your sidearm.)`
+					goto deathshot
+			label cotrans
+			action
+				set "asked about the Jibauni translators"
+			`	"I am not authorized to give you that information. Your questions?"`
+				goto meetingquestions
+			label lastisknown
+			`	"Yes. We know about you, <first>, Captain of the <ship>. How have you come here, for we did not detect any purple sparks?"`
+			choice
+				`	"I came through the wormhole. Do you mean to say you have not noticed it?"`
+					goto wormholeentry
+				`	"I came with a Jump Drive."`
+					to display
+						has "outfit (flagship installed): Jump Drive"
+					goto jumpdriveentry
+				`	"What do you mean, purple?"`
+					goto purple
+			label purple
+			`	"Oh, it does not matter. A side note, I believe. But, to rephrase my question, how did you enter our space?"`
```

Comment:
```suggestion
			`	 The creature bends it neck down so that it's massive face is level with your own, both yellow eyes staring directly in yours. Again the polite, incongruous voice drones out from the machine, "Oh, it does not matter. A side note, I believe. But, to rephrase my question, how did you enter our space?"`
```